ios/FluentUI.Demo/FluentUI.Demo/Demos/BadgeViewDemoController.swift

```diff
@@ -49,25 +51,38 @@ class BadgeViewDemoController: DemoController {
         let badge = BadgeView(dataSource: dataSource)
         badge.delegate = self
         badge.isActive = isEnabled
+        badges.append(badge)
         return badge
     }
 
     func addBadgeSection(title: String, style: BadgeView.Style, isEnabled: Bool = true, overrideColor: Bool = false) {
         addTitle(text: title)
-        for size in BadgeView.Size.allCases.reversed() {
-            let badge = createBadge(text: "Kat Larsson", style: style, size: size, isEnabled: isEnabled)
+        for sizeCategory in BadgeView.SizeCategory.allCases.reversed() {
+            let badge = createBadge(text: "Kat Larsson", style: style, sizeCategory: sizeCategory, isEnabled: isEnabled)
             if overrideColor {
                 if isEnabled {
-                    badge.backgroundColor = UIColor(colorValue: GlobalTokens.sharedColors(.purple, .primary))
-                    badge.selectedBackgroundColor = UIColor(colorValue: GlobalTokens.sharedColors(.darkTeal, .tint20))
-                    badge.labelTextColor = UIColor(colorValue: GlobalTokens.neutralColors(.grey94))
-                    badge.selectedLabelTextColor = UIColor(colorValue: GlobalTokens.neutralColors(.grey88))
+                    badge.tokenSet[.backgroundTintColor] = .uiColor {
+                        .init(light: GlobalTokens.sharedColor(.purple, .primary))
+                    }
```

Comment:
We shouldn't need the `.init` anymore, and should just be able to use the value from `GlobalTokens.sharedColor()`.